assignment.js

```diff
@@ -66,6 +73,33 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+//Alternative using Math functions
+let numbers = [57, 13, 4, 38]
+currentLargestNumber = Math.max(...numbers)
+
+console.log("Using Math functions: ")
+console.log(currentLargestNumber);
+
```

Comment:
@AtleLohrmann Nice work using the es6 spread operator syntax and using the Math.max method to return the largest value. This is a nice use of the library built-in tools.
Take a look at a different approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)